### PR TITLE
Feature/43-formulario-registro

### PR DESCRIPTION
**Pull request by ferandgal**
Formulario de registro finalizado, validado y refactorizado

### CODE REVIEW

File: src/components/Navbar.elements.js [hunk 1]
```diff
@@ -1,7 +1,9 @@
+import { DropdownButton } from 'react-bootstrap';
```
Comment (luibelzan) [inline src/components/Navbar.elements.js:1]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/park-inn-ispp/park-inn-frontend/pullRequest?prid=9172775)
```suggestion
import { DropdownButton } from "react-bootstrap";
```

File: src/components/GeneralButton/GeneralButton.elements.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import styled from 'styled-components'
```
Comment (luibelzan) [inline src/components/GeneralButton/GeneralButton.elements.js:1]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/park-inn-ispp/park-inn-frontend/pullRequest?prid=9172775)
```suggestion
import styled from "styled-components"
```

File: src/components/GeneralButton/GeneralButton.elements.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+import { colors } from '../../theme'
```
Comment (luibelzan) [inline src/components/GeneralButton/GeneralButton.elements.js:2]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/park-inn-ispp/park-inn-frontend/pullRequest?prid=9172775)
```suggestion
import { colors } from "../../theme"
```

File: src/Login/Logout.js [hunk 4]
```diff
@@ -7,6 +7,9 @@ const cookies = new Cookies();
 
 export default function Logout(){
     cookies.remove('AuthToken'); // determine if authorized, from context or however you're doing it
+    cookies.remove('user_mail');
```
Comment (luibelzan) [inline src/Login/Logout.js:10]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/park-inn-ispp/park-inn-frontend/pullRequest?prid=9172775)
```suggestion
    cookies.remove("user_mail");
```

File: src/Login/Logout.js [hunk 5]
```diff
@@ -7,6 +7,9 @@ const cookies = new Cookies();
...
+    cookies.remove("UserData");
+    cookies.remove('user_autorities');
```
Comment (luibelzan) [inline src/Login/Logout.js:12]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/park-inn-ispp/park-inn-frontend/pullRequest?prid=9172775)
```suggestion
    cookies.remove("user_autorities");
```

File: src/components/Formulario/Formulario.components.js [hunk 6]
```diff
@@ -0,0 +1,75 @@
+import styled from 'styled-components'
```
Comment (luibelzan) [inline src/components/Formulario/Formulario.components.js:1]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/park-inn-ispp/park-inn-frontend/pullRequest?prid=9172775)
```suggestion
import styled from "styled-components"
```

**Comment by ferandgal** [comment]
Pruebas de métricas de Codacy

**Pull request closed without merging**